packages/focuslock/src/react/__specs__/index.spec.js

```diff
@@ -3,15 +3,113 @@ import { mount } from 'enzyme'
 
 import FocusLock from '../index'
 
-describe('FocusLock', () => {
-  let props
+const children = (
+  <ul>
+    <li>
+      <a href="#">first</a>
+    </li>
+    <li>
+      <a href="#">second</a>
+    </li>
+    <li>
+      <a href="#">third</a>
+    </li>
+    <li>
+      <a href="#">forth</a>
+    </li>
+  </ul>
+)
 
+describe('FocusLock', () => {
   beforeEach(() => {
-    props = {
-      children: 'children'
-    }
+    document.activeElement.blur()
+  })
+
+  it('has defaultProps', () => {
```

Comment:
I have never tested default props as a structure.  I would maybe rather test the outcome that the existence of a defaultProp is meant to create.  What value have you found in testing this in the past?